README.md

```diff
@@ -5,45 +5,31 @@
 
 ## Features
 
-+ Automatic setup via one-liner (see section [Usage](#usage)), which: 
-
-    + Loads virtualenv, so you can start straight away.
-    
-    + Installs Invoke Bash tab-completion.
-    
-    + Installs app dependencies with help of [pip-tools](https://github.com/jazzband/pip-tools).
-    
-    + Initializes git repository and makes first commit.
-    
-    + Installs pre-commit and checks if it works.
-
-+ Configured .gitignore.
++ Prefilled .gitignore.
 
 + Configured Bandit.
 
 + Configured Flake8.
 
 + Configured isort.
 
-+ Configured logging.
++ Configured Black.
 
-+ Configured YAPF.
++ Configured mypy.
 
-+ Ready-to-go CI script.
++ Bitbucket Pipelines CI config.
```

Comment:
Would probably leave previous version. Not sure if binding to bitbucket is correct

Reply:
IMO as long as we have `bitbucket-pipelines.yml` in repo yep, it's correct. It's far from generic script :smile: